File: Beecrowd/3170 Bolinhas de Natal/main.c
#include <stdio.h>

int main(){
    int b, g, res;

    scanf("%d", &b);
    scanf("%d", &g);

    res = (g / 2) - b;

    if(res <= 0)
        printf("Amelia tem todas bolinhas!\n");
    else
        printf("Faltam %d bolinha(s)\n", res);

    return 0;
}

File: Beecrowd/3170 Bolinhas de Natal/main.py
b = int(input())
g = int(input())

res = (g // 2) - b # Divisão com (//) para retornar um numero inteiro

if res <= 0:
    print("Amelia tem todas bolinhas!")
else:
    print(f"Faltam {res} bolinha(s)")